manage.go

```diff
@@ -0,0 +1,84 @@
+//go:build manage
+// +build manage
+
+package main
+
+import (
+	"context"
+	"flag"
+	"os"
+
+	"github.com/DiscordGophers/dgobot/commands"
+
+	"github.com/bwmarrin/discordgo"
+	"github.com/peterbourgon/ff/v3"
+	"github.com/peterbourgon/ff/v3/ffcli"
+)
+
+var (
+	session *discordgo.Session
+	GuildID string
+)
+
+func main() {
+	fs := flag.NewFlagSet("dgbot", flag.ExitOnError)
+	token := fs.String("token", "", "Discord Authentication Token")
+	fs.StringVar(&GuildID, "guild-id", "", "Discord Guild ID")
+	cmd := ffcli.Command{
+		Name:    "dgobot manager",
+		FlagSet: fs,
+		Options: []ff.Option{ff.WithEnvVarPrefix("DG")},
+		Subcommands: []*ffcli.Command{
+			{
+				Name:       "add",
+				ShortUsage: "Add/upsert/overwrite commands",
+				Exec:       add,
+			},
+			{
+				Name:       "remove",
+				ShortUsage: "Remove/delete commands",
+				Exec:       remove,
+			},
+		},
+	}
+
+	err := cmd.Parse(os.Args[1:])
+	if err != nil {
+		panic(err)
+	}
+
+	session, err = discordgo.New("Bot " + *token)
+	if err != nil {
+		panic(err)
+	}
+	if err := session.Open(); err != nil {
+		panic(err)
+	}
+	defer session.Close()
```

Comment:
I don't think it's necessary to start the session here as gateway doesn't seem to be used.